### PR TITLE
Add docs for PersistentVolume last phase transition time

### PR DESCRIPTION
**Pull request by RomanBednar**
KEP: https://github.com/kubernetes/enhancements/pull/3796
Code: https://github.com/kubernetes/kubernetes/pull/116469

### CODE REVIEW

**Comment by RomanBednar** [comment]
@jsafrane please review from technical side

@katcosgrove I was able to update the PR since the deadline moved to 8th of August, can you please review it?

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -768,6 +768,23 @@ A volume will be in one of the following phases:
...
+volumes the phase is set to "Pending" and `LastPhaseTransitionTime` is set to
+current time.
+
+This field is not restricted and users can set any arbitrary timestamp for this
+field which will be kept until volume transitions phase again. If it does transition
+phase the user set value will be overwritten to current time by the API server.
+
+The timestamp can also be updated to reflect an earlier point in time relative
+to the current timestamp which can be useful for cases when system time setting
+might have been misconfigured and fixed later.
```
[the remark this answers is not in the document]
Comment (RomanBednar) [reply content/en/docs/concepts/storage/persistent-volumes.md]: Done, I've simplified it and added feature gate note.

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -768,6 +768,20 @@ A volume will be in one of the following phases:
 
 The CLI will show the name of the PVC bound to the PV.
 
+#### Phase transition timestamp
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Persistent volume status contains `LastPhaseTransitionTime` field which holds
```
Comment (tengqm) [inline content/en/docs/concepts/storage/persistent-volumes.md:775]:
```suggestion
Persistent volume status contains `lastPhaseTransitionTime` field which holds
```

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -768,6 +768,20 @@ A volume will be in one of the following phases:
...
+a timestamp of when the volume last transitioned its phase. For newly created
+volumes the phase is set to "Pending" and `LastPhaseTransitionTime` is set to
```
Comment (tengqm) [inline content/en/docs/concepts/storage/persistent-volumes.md:777]:
```suggestion
volumes the phase is set to "Pending" and `lastPhaseTransitionTime` is set to
```

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 4]
```diff
@@ -768,6 +768,20 @@ A volume will be in one of the following phases:
...
+current time.
+
+{{< note >}}
+You need to enable the `PersistentVolumeLastPhaseTransitionTime` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+to see `LastPhaseTransitionTime` field.
```
Comment (tengqm) [inline content/en/docs/concepts/storage/persistent-volumes.md:782]:
```suggestion
to see `lastPhaseTransitionTime` field.
```

**Pull request merged**